src/app/pages/header/header.component.ts

```diff
@@ -14,23 +13,31 @@ import { AppState } from '../../app.service';
   // We need to tell Angular's Dependency Injection which providers are in our app.
   providers: [],
   // Our list of styles in our component. We may add more to compose many styles together
-  styleUrls: [ './header.component.css' ],
+  styleUrls: [ './header.component.scss' ],
   // Every Angular template is first compiled by the browser before Angular runs it's compiler
   templateUrl: './header.component.html'
 })
 export class HeaderComponent implements OnInit {
   // Set our default values
   public localState = { value: '' };
+  public iLikeTrainsLogo = 'assets/img/I-LIKE-TRAINS.jpg';
+  public name = 'Angular 2 Webpack Starter';
+  public url = 'https://localhost:3000';
   // TypeScript public modifiers
   constructor(
     public appState: AppState
   ) {}
 
   public ngOnInit() {
     console.log('hello `Home` component');
+    this.logData('onInit');
     // this.title.getData().subscribe(data => this.data = data);
   }
 
+  private logData(msg: string) {
+    console.log(msg)
+  }
+
   public submitState(value: string) {
```

Comment:
Tslint warning:
`
WARNING in ./src/app/pages/header/header.component.ts
[41, 3]: Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after constructors.
[38, 21]: Missing semicolon

 @ ./src/app/pages/header/index.ts 1:0-35
 @ ./src/app/app.module.ts
 @ ./src/app/index.ts
 @ ./src/main.browser.ts
 @ multi (webpack)-dev-server/client?http://localhost:3000 ./src/main.browser.ts
`